### PR TITLE
Split Article Web Development in Python

### PR DESCRIPTION
**Pull request by Rafase282**
Splits the article, this one didn't have missing sections or To-Dos.

### CODE REVIEW

**Comment by theflametrooper** [comment]
:heavy_plus_sign:  `Python-Web-Framawork-Usage.md`
:heavy_minus_sign:  `Python-Web-Framework-Usage.md`

File: Web-Development-in-Python.md [hunk 1]
```diff
@@ -6,48 +6,10 @@
 
 [Next](Install-Django-Flask)
 
+- [Web Frameworks in Python](Python-Web-Frameworks)
+- [Web Applications using Python](Python-Web-Applications)
+- [Web Frameworks and What They Do For You](Python-Web-Framawork-Usage)
```
Comment (theflametrooper) [inline Web-Development-in-Python.md]: ~~Python-Web-Framawork-Usage~~ Python-Web-Framework-Usage

File: Python-Web-Framawork-Usage.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Web Frameworks and What They Do For You
+
+We have used the word `framework` quite librarily in the earlier sections. You might or might not be familiar with what that is. Nonetheless, we would discuss what Python Web Frameworks do for you, out-of-the-box.
```
Comment (alayek) [inline Python-Web-Framawork-Usage.md]: > We have used the word `framework` quite librarily in the earlier ~~sections~~ articles.

**Comment by alayek** [comment]
Barring the issues mentioned in line note, 

# LGTM :+1:

**Comment by Rafase282** [comment]
Changes made.

**Comment by theflametrooper** [comment]
Thanks @Rafase282 !

**Pull request merged**